documentation/asciidoc/computers/raspberry-pi/boot-http.adoc

```diff
@@ -0,0 +1,50 @@
+== HTTP Boot
+
+The network install feature uses HTTP over ethernet to boot the device into the xref:getting-started.adoc#using-raspberry-pi-imager[Raspberry Pi Imager] application.
+
+In addition to network install you can explicitly boot your device with files downloaded via HTTP with xref:raspberry-pi.adoc#BOOT_ORDER[boot mode] `7`. You can still use this even if xref:raspberry-pi.adoc#NET_INSTALL_ENABLED[network install / keyboard detection on boot is disabled].
+
+You could for example add this to your `BOOT_ORDER` as a fall back boot method or put it behind a GPIO conditional to initiate HTTP download from your own server when a GPIO is pulled low. e.g.
+
+```
+[gpio7=0]
+BOOT_ORDER=0xf7
+HTTP_HOST=downloads.raspberrypi.org
+NET_INSTALL_ENABLED=0
+```
```

Comment:
There seems to be a lack of context for this code-block (so people might thing it's something they'd add to `/boot/config.txt`?). Could say something like "For example, if you added the following to your eeprom config..."